user-docs/latest/content-manager/configuring-view-of-content-type.md

```diff
@@ -43,3 +59,56 @@ Note also that relational fields have a couple limitations when it comes to sort
 - Sorting cannot be enabled for relational fields which display several fields.
 - Relational fields cannot be set as default sort.
 :::
+
+## Configuring the edit view
+
+In the edit view of a content type, in the right side of the interface, a **Configure the view** button is displayed. It allows to access the configurations that can be set for the edit view of the content type, such as the entry title, and the display of the fields of the content type, including the relational ones.
```

Comment:
Structure seems a little odd but ran it through a grammar check to be sure and it passed so :shrug: just a comment, no change needed.